src/server/server.ts

```diff
@@ -267,6 +267,7 @@ app.get('/:articleId(*)/live-blocks', liveBlocks);
 app.get('/healthcheck', (_req, res) => res.send("Ok"));
 
 app.get('/favicon.ico', (_, res) => res.status(404).end());
+app.get('/fontSize.css', (_, res) => res.status(404).end());
```

Comment:
This is fine, but we could use `501 Not Implemented` instead here? Practical upshot is the same, but the semantics are nicer (and it's fun to use less-used response codes :man_shrugging:)

```suggestion
app.get('/fontSize.css', (_, res) => res.status(501).end());
```

Reply:
Unfortunately servers are **required** to support `GET`, and therefore **must not** return 501 for `GET` requests, as detailed [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/501).

Reply:
Ah nevermind!